packages/ui/components/input-datepicker/test/lion-input-datepicker.test.js

```diff
@@ -323,6 +323,29 @@ describe('<lion-input-datepicker>', () => {
         expect(elObj.calendarEl.maxDate).to.equal(myMaxDate);
       });
 
+      it('does not converts MaxDate validator to "maxDate" property if validator type other then "error"', async () => {
```

Comment:
Would be a good practice to test for all other validator types other than "error", besides "warning", wdyt?

Reply:
On lion we only have `error`.